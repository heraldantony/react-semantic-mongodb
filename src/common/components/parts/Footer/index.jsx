/**
 * @flow
 */
import React from "react";
import { Grid, Header, Icon } from "semantic-ui-react";
import { StyledFooter, StyledFooterInner } from "./style";

const Footer = () => {
  return (
    <StyledFooter>
      <StyledFooterInner>
        <Grid relaxed>
          <Grid.Row verticalAlign="middle">
            <Grid.Column width={12} mobile={16}>
              <a href="https://github.com/heraldantony/reactsemanticmongo">
                <Header as="h3" inverted>
                  <Icon name="github" />
                  <Header.Content>
                    <Header.Subheader>React Web App</Header.Subheader>
                  </Header.Content>
                </Header>
              </a>
            </Grid.Column>
          </Grid.Row>
        </Grid>
      </StyledFooterInner>
    </StyledFooter>
  );
};

export default Footer;
