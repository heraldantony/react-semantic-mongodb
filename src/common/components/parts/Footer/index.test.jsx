import React from 'react'
import Footer from 'components/parts/Footer'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Footer component', () => {
	it('same snapshot', () => {
		const component = shallow(<Footer />)
		expect(toJson(component)).toMatchSnapshot()
	})
})
