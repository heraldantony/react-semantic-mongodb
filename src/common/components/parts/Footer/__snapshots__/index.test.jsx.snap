// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component same snapshot 1`] = `
<style__StyledFooter>
  <style__StyledFooterInner>
    <Grid
      relaxed={true}
    >
      <GridRow
        verticalAlign="middle"
      >
        <GridColumn
          mobile={16}
          width={12}
        >
          <a
            href="https://github.com/Metnew/suicrux"
          >
            <Header
              as="h3"
              inverted={true}
            >
              <Icon
                as="i"
                name="github"
              />
              <HeaderContent>
                Suicrux
                <HeaderSubheader>
                  Starter for 
                  <s>
                    blade
                  </s>
                   bleeding-edge runners
                </HeaderSubheader>
              </HeaderContent>
            </Header>
          </a>
        </GridColumn>
      </GridRow>
    </Grid>
  </style__StyledFooterInner>
</style__StyledFooter>
`;
